framework_tvb/tvb/core/neotraits/_h5core.py

```diff
@@ -273,6 +276,7 @@ def __repr__(self):
 
 class ViewModelH5(H5File):
 
+    # TODO  it will be good to be able to just call with H5File.from_file(h5_path) as f for ViewModelH5 also
```

Comment:
@popaula937 I would like your opinion if this can be achieved or not. I think it can, then we should create a dedicated task for this TODO